analyzers/tests/SonarAnalyzer.TestFramework.Test/Verification/VerifierTest.cs

```diff
@@ -513,27 +514,31 @@ public void Verify_AutogenerateConcurrentFiles()
     public void Verify_TestProject()
     {
         var builder = new VerifierBuilder<DoNotWriteToStandardOutput>() // Rule with scope Main
-            .AddSnippet("public class Sample { public void Main() { System.Console.WriteLine(); } }");
-        builder.Invoking(x => x.Verify()).Should().Throw<AssertFailedException>();
-        builder.AddTestReference().Invoking(x => x.Verify()).Should().NotThrow("Project references should be recognized as Test code.");
+            .AddSnippet("class Sample { void Main() { System.Console.WriteLine(); } }  // Noncompliant");
+        builder.Invoking(x => x.Verify()).Should().NotThrow();
+        builder.AddTestReference().Invoking(x => x.Verify()).Should().Throw<AssertFailedException>("project references should be recognized as Test code").WithMessage("""
+            There are differences for CSharp7 snippet0.cs:
+              Line 1: Missing expected issue
+            """);
```

Comment:
I'm a bit brain-fogged at this point of the review 😄 so I might be missing something here: 
it seems to me that the assertion changed for Verify and doesn't seem right, please send me a zoom link whenever you reach this point so we can discuss it